bazel/external/curl.BUILD

```diff
@@ -0,0 +1,248 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Why not use `cmake_external` for curl?

Reply:
This PR predated `cmake_external`, so yeah, might be worth a try to simplify with that.